wri/gfw
# [FLAG-934] Conditionally hide 2 widgets for European countries

## Overview

Adequate plantation data for European countries does not exist and therefore two of our widgets show misleading data (`treeCoverGainOutsidePlantations`, `treeLossPlantations`). So, we would like to conditionally hide the following widgets when a European  administrative areas (FAO Europe) is selected on the map or dashboard:


## Testing

-  Open the test environment
-  Go to the Dashboard or to the Map.
- Select a Europe FAO country
- Check if the denied widgets are hidden.
